src/middlewared/middlewared/api/v25_04_0/auth.py

```diff
@@ -22,7 +22,7 @@ class AuthLegacyUsernamePassword(BaseModel):
     password: Secret[str]
 
 
-class AuthLegacyPasswordLoginArgs(AuthLegacyUsernamePassword):
+class AuthLoginArgs(AuthLegacyUsernamePassword):
```

Comment:
Why are we renaming these? It seems out of scope.

Reply:
NEP says that class names should match method names